modules/monitoring-resizing-a-persistent-storage-volume.adoc

```diff
@@ -0,0 +1,203 @@
+Orphaning the pods recreates the StatefulSet immediately and automatically updates the size of the volumes mounted in the pods with the new PVC settings.
+No service disruption occurs during this process. 
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* *If you are configuring core {product-title} monitoring components*:
+** You have access to the cluster as a user with the `cluster-admin` role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
+* *If you are configuring components that monitor user-defined projects*:
+** You have access to the cluster as a user with the `cluster-admin` role, or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+** You have created the `user-workload-monitoring-config` `ConfigMap` object.
+
+.Procedure
+
+. Edit the `ConfigMap` object:
+** *To configure a PVC for a component that monitors core {product-title} projects*:
+.. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+.. Add a new storage size for the PVC configuration for the component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    <component>: <1>
+      volumeClaimTemplate:
+        spec:
+          storageClassName: <storage_class> <2>
+          resources:
+            requests:
+              storage: <amount_of_storage> <3>
+----
+<1> Specify the core monitoring component.
+<2> Specify the storage class.
+<3> Specify the new size for the storage volume.
++
+The following example configures a PVC that changes the local persistent storage to 100 gigabytes for the Prometheus instance that monitors core {product-title} components:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: local-storage
+          resources:
+            requests:
+              storage: 100Gi
+----
++
+The following example changes a PVC that changes local persistent storage for Alertmanager to 40 gigabytes:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    alertmanagerMain:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: *local-storage*
+          resources:
+            requests:
+              storage: 40Gi
+----
+
+** *To configure a PVC for a component that monitors user-defined projects*:
++
+[NOTE]
+====
+You can resize the volumes for the Alertmanager, Thanos Ruler and Prometheus instances that monitor user-defined projects.
```

Comment:
dedicated Alertmanager for user workload monitoring is since 4.11, by default, the dedicated Alertmanager is not enabled , since this doc is for 4.9+, we don't need to mention Alertmanager here